### PR TITLE
Fix separation

### PR DESCRIPTION
**Pull request by rewin123**
- Allow to hide top and bottom menu with EditorUiShow state
- Remove duplication of adding Selected plugin

### CODE REVIEW

File: crates/editor_ui/src/menu_toolbars.rs [hunk 1]
```diff
@@ -36,11 +36,13 @@ impl Plugin for BottomMenuPlugin {
 
         app.add_systems(
             Update,
-            bottom_menu.before(EditorLoadSet).in_set(EditorSet::Editor),
+            bottom_menu.before(EditorLoadSet).in_set(EditorSet::Editor)
+                .run_if(in_state(EditorState::Editor).and_then(in_state(ShowEditorUi::Show))),
```
[the remark this answers is not in the document]
Comment (rewin123) [reply crates/editor_ui/src/menu_toolbars.rs:40]: If state is ShowEditorUi::Hide - all editor ui hides
If state is ShowEditorUi::Show - all editor ui shows 

Its good for integrating space_editor just as debuger, which I use in one my small project

[the remark this answers is not in the document]
Comment (naomijub) [reply crates/editor_ui/src/menu_toolbars.rs:40]: So this is only available as debugger? Can't we trigger enabling and disabling?

[the remark this answers is not in the document]
Comment (rewin123) [reply crates/editor_ui/src/menu_toolbars.rs:40]: We can
image

[the remark this answers is not in the document]
Comment (naomijub) [reply crates/editor_ui/src/menu_toolbars.rs:43]: So, it is done in  settings right?

[the remark this answers is not in the document]
Comment (rewin123) [reply crates/editor_ui/src/menu_toolbars.rs:43]: No. I was only doing it for code style manipulation of the editor display. Are you suggesting to make it an option - to hide the whole UI in game mode?

[the remark this answers is not in the document]
Comment (naomijub) [reply crates/editor_ui/src/menu_toolbars.rs:43]: Not right now, just wanted to understand how to use it

[the remark this answers is not in the document]
Comment (naomijub) [reply crates/editor_ui/src/menu_toolbars.rs:43]: Changing the UI before the design is done will panic the designer hahaha

**Review by naomijub — APPROVED** [review]

File: crates/editor_ui/src/menu_toolbars.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (rewin123) [reply crates/editor_ui/src/menu_toolbars.rs:43]: Huh. Well, I think we need to improve the debagging of games. So checkboxes have been added to show all entities/components. It's a small step to make space_editor not only a thing in itself, but also a third party handy debugger like bevy_editor_pls

**Pull request merged**